cna-memory-considerate-animation/components/Layout/index.js

```diff
@@ -31,13 +31,8 @@ const Layout = ({ children, clientHintDeviceMemory, hookMemoryStatus }) => {
       <Head>
         <title>Adaptive Animation</title>
```

Comment:
CNA: GitHub link in this demo should link back to the original implementation we adapted (right now it links to your personal GitHub :))

![image](https://user-images.githubusercontent.com/110953/68132078-ee882d80-fed2-11e9-80c6-376e29083f6d.png)

Reply:
Let me change the link to the original implementation.
Should we link to GitHub repo or deployed URL?

Reply:
I think we should link to the repo

Reply:
It's done so.

Reply:
Please check it.